docker/worker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:23.10
+
+RUN apt update -y && apt upgrade -y && apt install -y \
+  git vim python3 python3-distutils make g++ # clang llvm
+
+RUN useradd --create-home --shell /bin/bash worker
+
+WORKDIR /home/worker
+COPY run_workers.sh .
+RUN chmod +x run_workers.sh
+
```

Comment:
It seems to me that with this setup it would be possible to start the same image multiple times. Am I right?